server/service/orbit.go

```diff
@@ -998,11 +998,31 @@ func (svc *Service) SaveHostSoftwareInstallResult(ctx context.Context, result *f
 			return ctxerr.Wrap(ctx, err, "get host software installation result information")
 		}
 
+		// Self-Service packages will have a nil author for the activity.
```

Comment:
Can you add a test for the change in this file?

Reply:
This is being tested in the added test in `integration_enterprise_test.go`.